collect-intent.rst

```diff
@@ -0,0 +1,207 @@
+Using ODK Collect with External Apps
+======================================
+
+.. _launch-collect:
+
+Launching ODK Collect from External Apps
+------------------------------------------
+
+:doc:`collect-guide` allows us to open several of its activities from another app. You can open a specific form or lists of empty forms, saved forms, finalized forms or sent forms. You can also build your own app that interacts with ODK Collect through intents. On the other side, you can also open other apps from ODK Collect.
+
+This doc describes how to launch ODK Collect and open its activities directly from an external app.
+
+.. _about-intents:
+
+Understanding Intents
+~~~~~~~~~~~~~~~~~~~~~~~
+
+An Intent is a messaging object you can use to request an action from another app component. 
+
+For example:
```

Comment:
I think this example confuses things a little bit because this document is about using intents to work with OTHER applications. I would just say "An Intent is a messaging object you can use to request an action from another app component. For more details refer to `these Android docs <https://developer.android.com/guide/components/intents-filters.html>`_."